diff --git a/Lib/LibStatic/CMakeLists.txt b/Lib/LibStatic/CMakeLists.txt
index c116ee5..165e1e8 100644
--- a/Lib/LibStatic/CMakeLists.txt
+++ b/Lib/LibStatic/CMakeLists.txt
@@ -27,8 +27,8 @@ IF(WIN32 AND NOT UNIX)
 
 ELSE(WIN32 AND NOT UNIX)
 	target_link_libraries (RakNetLibStatic ${RAKNET_LIBRARY_LIBS})
-	INSTALL(TARGETS RakNetLibStatic DESTINATION ${RakNet_SOURCE_DIR}/Lib/RakNetLibStatic)
-	INSTALL(FILES ${ALL_HEADER_SRCS} DESTINATION ${RakNet_SOURCE_DIR}/include/raknet)
+	INSTALL(TARGETS RakNetLibStatic DESTINATION lib)
+	INSTALL(FILES ${ALL_HEADER_SRCS} DESTINATION include/raknet)
 ENDIF(WIN32 AND NOT UNIX)
 
 
