diff --git a/GeneratedSaxParser/src/GeneratedSaxParserUtils.cpp b/GeneratedSaxParser/src/GeneratedSaxParserUtils.cpp
index 1f9a3ee..d151e9a 100644
--- a/GeneratedSaxParser/src/GeneratedSaxParserUtils.cpp
+++ b/GeneratedSaxParser/src/GeneratedSaxParserUtils.cpp
@@ -1553,7 +1553,7 @@ namespace GeneratedSaxParser
 #if defined(COLLADABU_OS_WIN) && !defined(__MINGW32__)
         return _isnan( value ) ? true : false;
 #else
-#ifdef isnan
+#if defined(isnan) || defined(__APPLE__)
         return isnan( value );
 #else
         return std::isnan(value);
